package.nls.json

```diff
@@ -50,6 +50,7 @@
     "AWS.explorerNode.addRegion.tooltip": "Click to add a region to view functions...",
     "AWS.explorerNode.lambda.noFunctions": "[No Functions found]",
     "AWS.explorerNode.cloudFormation.noFunctions": "[no functions in this CloudFormation]",
+    "AWS.explorerNode.cloudformation.noStacks": "[No Stacks found]",
```

Comment:
Suuuuuuuuuuuuuuuuuuper tiny nit/thought: I'm not huge on the fact that it's the string's responsibility to add the brackets (instead of the placeholder node itself). That being said, it's probably not worth having another class that extends `PlaceholderNode` just to wrap error messages in brackets.